pkg/controller/nodenetworkstate/nodenetworkstate_controller.go

```diff
@@ -24,9 +26,21 @@ import (
 
 var (
 	log                     = logf.Log.WithName("controller_nodenetworkstate")
-	nodenetworkstateRefresh = 5 * time.Second
+	nodenetworkstateRefresh time.Duration
 )
 
+func init() {
```

Comment:
Do this at main and pass to constructed controller, this is very implicit.

Reply:
It is not clear how would we pass any extra data from main() down. Also, it would require us to either create a singleton for it or pass it through several functions all the way down here. I'd personally prefer init() in this case. But if you think passing it down would be better (totally fair, makes sense to keep single interface between app and the environment, i won't fight for init()).

Reply:
Puff I see a lot of dependency injections let's just do at init (but I still don't like them)